image.yaml

```diff
@@ -36,11 +36,11 @@ modules:
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_730_CR4
+                  ref: EAP_730_SP01
```

Comment:
for reference, `EAP_730_SP01` & `EAP_730_CR4` are identical: https://github.com/jboss-container-images/jboss-eap-modules/compare/EAP_730_CR4...EAP_730_SP01